Objective-C/CBLFileLogger.mm

```diff
@@ -47,15 +47,19 @@ - (instancetype) initWithDefault {
 
 
 - (NSString*) defaultDirectory {
-    NSArray* paths = NSSearchPathForDirectoriesInDomains(NSCachesDirectory, NSUserDomainMask, YES);
-    NSString* path = paths[0];
 #if !TARGET_OS_IPHONE
     NSString* bundleID = [[NSBundle mainBundle] bundleIdentifier];
-    if (!bundleID)
-        return [NSFileManager.defaultManager currentDirectoryPath]; // last-ditch default for non-apps
-    path = [path stringByAppendingPathComponent: bundleID];
+    if (bundleID) {
+        NSArray* paths = NSSearchPathForDirectoriesInDomains(NSLibraryDirectory, NSUserDomainMask, YES);
+        return [paths[0] stringByAppendingPathComponent:
+                [bundleID stringByAppendingPathComponent: @"/CouchbaseLite"]];
```

Comment:
this will return ~/Library/<bundleID>/CouchbaseLite ?? but we need ~/Library/Logs/<bundleID>/CouchbaseLite ??
```suggestion
                [NSString stringWithFormat: @"Logs/%@/CouchbaseLite", bundleID]];
```

Reply:
Thanks and fixed.